workbench/src/main/main.js

```diff
@@ -9,35 +9,37 @@ import {
   ipcMain
 } from 'electron';
 
+import BASE_URL from './baseUrl';
 import {
   createPythonFlaskProcess,
   getFlaskIsReady,
-  shutdownPythonProcess
+  shutdownPythonProcess,
 } from './createPythonFlaskProcess';
 import findInvestBinaries from './findInvestBinaries';
-import setupDownloadHandlers from './setupDownloadHandlers';
-import setupDialogs from './setupDialogs';
-import setupContextMenu from './setupContextMenu';
+import { ipcMainChannels } from './ipcMainChannels';
+import ELECTRON_DEV_MODE from './isDevMode';
+import { getLogger } from './logger';
+import menuTemplate from './menubar';
+import pkg from '../../package.json';
+import { settingsStore, setupSettingsHandlers } from './settingsStore';
+import { setupBaseUrl } from './setupBaseUrl';
 import setupCheckFilePermissions from './setupCheckFilePermissions';
 import { setupCheckFirstRun } from './setupCheckFirstRun';
 import { setupCheckStorageToken } from './setupCheckStorageToken';
+import setupContextMenu from './setupContextMenu';
+import setupDialogs from './setupDialogs';
+import setupDownloadHandlers from './setupDownloadHandlers';
+import setupGetElectronPaths from './setupGetElectronPaths';
+import setupGetNCPUs from './setupGetNCPUs';
 import {
+  setupInvestLogReaderHandler,
   setupInvestRunHandlers,
-  setupInvestLogReaderHandler
 } from './setupInvestHandlers';
-import setupGetNCPUs from './setupGetNCPUs';
+import { setupIsNewVersion } from './setupIsNewVersion';
 import setupOpenExternalUrl from './setupOpenExternalUrl';
 import setupOpenLocalHtml from './setupOpenLocalHtml';
-import { settingsStore, setupSettingsHandlers } from './settingsStore';
-import setupGetElectronPaths from './setupGetElectronPaths';
 import setupRendererLogger from './setupRendererLogger';
-import { ipcMainChannels } from './ipcMainChannels';
-import menuTemplate from './menubar';
-import ELECTRON_DEV_MODE from './isDevMode';
-import BASE_URL from './baseUrl';
-import { getLogger } from './logger';
-import i18n from './i18n/i18n';
-import pkg from '../../package.json';
+
```

Comment:
The only imports added to this file are `setupBaseUrl` and `setupIsNewVersion`. All the other changes are just reorganization to make the file a bit easier to read.